app/src/main/res/values-fr/strings.xml

```diff
@@ -255,4 +255,5 @@ L'information que vous entrez est directement ajoutée à OpenStreetMap sous vot
     <string name="upload_server_error">"Erreur de connexion lors de l'envoi des réponses. Essayez plus tard."</string>
     <string name="quest_generic_hasFeature_no">"Non"</string>
     <string name="quest_generic_hasFeature_yes">"Oui"</string>
+    <string name="quest_bikeParkingCapacity_title">"Combien de vélos peut-on garer ici ?"</string>
```

Comment:
By the way, I usually do not export translations other than the source translation (English) to POEditor, translations should be made only in POEditor as translations here will be overwritten.